dev/tests/integration/framework/Magento/TestFramework/Workaround/Cleanup/StaticProperties.php

```diff
@@ -187,7 +187,8 @@ public static function backupStaticVariables()
             function ($classFile) {
                 return StaticProperties::_isClassInCleanableFolders($classFile)
                     // phpcs:ignore Magento2.Functions.DiscouragedFunction
-                    && strpos(file_get_contents($classFile), ' static ')  > 0;
+                    && strpos(file_get_contents($classFile), ' static ')  > 0
+                    && strpos($classFile, 'TestFramework')  === -1;
```

Comment:
It would be better to check strpos before getting file content

Reply:
@ihor-sviziev updated